01-JS-DOM-APIs/code.js

```diff
@@ -0,0 +1,104 @@
+function show(){
+                let obj = document.getElementById("hidden");
```

Comment:
check indentation here,
you can assign the styles you need (display: block or display: none) to classes, and instead of editing obj.style, add or remove that class